imgui_plot.h

```diff
@@ -1102,14 +1105,32 @@ namespace ImGui
             }
             float const * const y = y_data_bottoms ? y_data_bottoms : y_data;
 
+            std::vector<ImVec2> v;
```

Comment:
No STL per the primary authors wishes to keep things "fast" and low level for GUI applications.

Reply:
Yeah I was just making things work. I'll get rid of it :smile: